canonical-web-and-design/ubuntu.com
# Filter languages for related pages

## Done

- Filter languages for related pages
- Add test for engage pages

## QA

- Check with the demo that https://ubuntu-com-8447.demos.haus/engage/vmware-to-charmed-openstack/thank-you does not show languages other than English
- Check that the github checks pass


## Issue / Card

Fixes #8440
